tests/fetchhead/nonetwork.c

```diff
@@ -508,7 +508,7 @@ void test_fetchhead_nonetwork__create_with_multiple_refspecs(void)
 		int i;
 		struct prefix_count prefix_counts[] = {
 			{"refs/notes/", 0, 1},
-			{"refs/heads/", 0, 12},
+			{"refs/heads/", 0, 13},
```

Comment:
Ugh, these tests that test the number of refs are so annoying when you add a new one, aren't they?  This has bitten me many times.  Thanks for updating them all.  🙏